samples/IISSample/web.config

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0"?>
```

Comment:
this sample is not used for testing NTLM though so we don't need to set `forwardWindowsAuthToken` to true